hackair-v2-advanced.ino

```diff
@@ -123,31 +127,32 @@ void setup() {
         std::unique_ptr<char[]> buf(new char[size]);
 
         configFile.readBytes(buf.get(), size);
-        DynamicJsonBuffer jsonBuffer;
-        JsonObject& json = jsonBuffer.parseObject(buf.get());
-        json.prettyPrintTo(Serial);
-        if (json.success()) {
+        DynamicJsonDocument jsonDoc(1024);
```

Comment:
Not sure about the 1024 here, but this is how it's done in the "How To Upgrade To 6.x" documentation.